plugins/collector/api/collector.go

```diff
@@ -27,7 +27,7 @@ import (
 //   Init()     Initial stage: Init plugin by config
 //    ||
 //    \/
-//   Prepare()   Preparing stage: Prepare the collector, such as build connection with SkyWalking javaagent.
+//   Init()   Preparing stage: Init the collector, such as build connection with SkyWalking javaagent.
```

Comment:
Why duplicated `Init`? And `such as build connection with SkyWalking javaagent.` is wrong. I think you mean, set up the gRPC server for the agents?

Reply:
it is a mistake when use goland refactor